src/components/Button.tsx

```diff
@@ -15,15 +15,17 @@ const styles = () =>
     },
   });
 
+  // TODO: @wangannie figure out how onclick would work in relation to linking to avoid confusion
 interface MainButtonProps {
   classes: { button: string };
   label: string;
+  onClick?: (event: React.MouseEvent) => void;
```

Comment:
I need this `onClick` for the `handleSubmit` on the add customer form, but I know that in most other places, these buttons are used with `<Link>` tags instead, so I worry about this being confusing. Maybe there's a way to consolidate to `onClick` and wrap the Button in `Link` tags here, then just pass in the destination through `onClick`? Or I'm overthinking it and this is fine :~)

Reply:
I think this is fine but i'll take another look in the morning (brain is kinda fried)